docs/source/index.rst

```diff
@@ -0,0 +1 @@
+Rest-API Plugin <rest-api>
```

Comment:
Not quite, you need to make it a recognized TOC for Sphinx.

```suggestion
.. toctree::
   :maxdepth: 1
   :caption: Project Documentation
   :hidden:

   Rest-API Plugin <rest-api>
```

And include a portion of the home page index from AaC Core.

Or convert the plugin md page to rst so that it can just be the landing page. The index/landing pages of Sphinx need to be rst, and other pages can be md with minimal issue. 

Remember to review the local documentation build for behavior and looks. 

Check the index pages in core repo for the needed configurations.